awscli/clidriver.py

```diff
@@ -105,7 +107,13 @@ def create_clidriver():
 
 
 def _get_distribution_source():
-    return 'bundle'
+    metadata_file = os.path.join(os.environ['AWS_DATA_PATH'],
```

Comment:
Instead of using `AWS_DATA_PATH` here, we should probably use the path to the `awscli/data` directory. The `AWS_DATA_PATH` indicates where to look for additional models to load so it is possible for users to set it and therefore this will not point to this file. Typically if we want to rely on the `AWS_DATA_PATH` env var, we should probably go through a botocore loader, but in this case I'm not sure if we need to be relying on it.